server/routes/prisons/sessionTemplates/addSessionTemplateController.ts

```diff
@@ -61,6 +61,9 @@ export default class AddSessionTemplateController {
           endTime: req.body.endTime,
         },
         visitRoom: req.body.visitRoom,
+        categoryGroupReferences: [],
```

Comment:
Defaulting these to empty here for now - can be populated later once we have the 'groups' functionality